torch/csrc/jit/backends/backend_debug_handler.h

```diff
@@ -114,42 +114,32 @@ using DebugHandleType = int64_t;
 using BackendDebugInfoMapType =
     std::unordered_map<DebugHandleType, DebugInfoTuple>;
 
+using NodeToDebugHandle = std::unordered_map<Node*, DebugHandleType>;
```

Comment:
I think it'd be cleaner to have these in backend_detail.h

My logic was that since they're mainly for easy reading for the backend developers to have a nice cimple signature for ```BackendPreprocessFunction```

Reply:
OK that makes sense.